src/Accounts/Authenticators/Identity/ITokenCacheOptions.cs

```diff
@@ -27,7 +29,7 @@ public class TokenCachePersistenceOptions
 
         /// <summary>
         /// If set to true the token cache may be persisted as an unencrypted file if no OS level user encryption is available. When set to false the token cache
-        /// will throw a <see cref="UnsafeAllowUnencryptedStorage"/> 
+        /// will throw a <see cref="CredentialUnavailableException"/> in the event no OS level user encryption is available.
```

Comment:
Please see https://github.com/Azure/azure-powershell/pull/18663/files, Yuzhi modified it -> will throw a <see cref="UnsafeAllowUnencryptedStorage"/> 
why you roll back her change?

Reply:
Because her change is wrong. What is the mean of "a token cache will throw a bool property"? Here should be an exception class of course, and the original exception class can be found with using a namespace at the top.